Sources/Testing/Testing.docc/MigratingFromXCTest.md

```diff
@@ -440,11 +438,10 @@ functionality called _confirmations_ which can be used to implement these tests.
 Instances of ``Confirmation`` are created and used within the scope of the
 function ``confirmation(_:expectedCount:fileID:filePath:line:column:_:)``.
 
-Confirmations function similarly to XCTest's expectations, however they do not
+Confirmations function similarly to the expectations of XCTest; however, they don't
```

Comment:
This reads like XCTest is expecting something of me. Expectations in XCTest are a feature thereof.

Reply:
Was mainly trying not to make XCTest possessive. Is there a better way to do this and retain the intent?

Reply:
"irmations function similarly to the expectations API of XCTest" or something like that?